.github/workflows/spelling.yml

```diff
@@ -0,0 +1,25 @@
+name: Spell checking
+on:
+  pull_request_target:
+  push:
+  issue_comment:
+
+jobs:
+  build:
+    name: Spell checking
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout-merge
+      if: "contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
+      with:
+        ref: refs/pull/${{github.event.pull_request.number}}/merge
+        fetch-depth: 5
+    - name: checkout
+      if: "!contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
+      with:
+        fetch-depth: 5
+    - uses: check-spelling/check-spelling@prerelease
+      with:
+        experimental_apply_changes_via_bot: 1
```

Comment:
I like the idea of less hassle. Can this be done _only_ for PRs, though?

Reply:
This feature (in preview, and 0.0.18 once released) is only available for PRs. Specifically while GitHub does have a `comment` event, it doesn't get sent to GitHub workflows, so the only chances I have to respond to comments are `issue_comment` (which practically means a PR comment, since the bot will not respond to a thing that doesn't have a proper comment from itself and a branch...) and `pull_request_review_comment` (which I am not handling now -- and since the bot's comment is top level, it doesn't make sense for a user to somehow reply to the bot in the wrong place).

I am planning to offer a way for repositories / commits to ask the bot to do the work.

Something like a commit message:

```
feat: Add support for blah-parsing

@check-spelling-bot: update expect
```

Which would then be handled by the `push` handler. I hope to be able to do this for 0.0.19. The precise way this will work is still entirely in the air. As noted elsewhere, I'm still working on the wiring to support it, and I haven't even started thinking about the command structure. (In a last minute thing for 0.0.18, I added support for the bot also updating the `excludes.txt` file in addition to `expect.txt`.)